react-with-arrays/react-with-arrays_journal-app-entries-array/src/components/EntriesSection/index.js

```diff
@@ -17,23 +46,17 @@ export default function EntriesSection() {
         </Tab>
       </Tabs>
       <div className="entries-section__entries">
-        <Entry
-          date="Feb 27, 2028"
-          motto="Thats life in the city"
-          notes="Si sine causa? quae fuerit causa, mox videro; interea hoc tenebo, si mihi. Et quidem se repellere, idque instituit docere sic omne animal, simul atque."
-        />
-        <Divider />
-        <Entry
-          date="Feb 26, 2028"
-          motto="Thats life in the city"
-          notes="Si sine causa? quae fuerit causa, mox videro; interea hoc tenebo, si mihi. Et quidem se repellere, idque instituit docere sic omne animal, simul atque."
-        />
-        <Divider />
-        <Entry
-          date="Feb 25, 2028"
-          motto="Thats life in the city"
-          notes="Si sine causa? quae fuerit causa, mox videro; interea hoc tenebo, si mihi. Et quidem se repellere, idque instituit docere sic omne animal, simul atque."
-        />
+        {entries &&
+          entries.map((entry, index) => (
+            <>
+              <Entry
+                date={entry.date}
```

Comment:
Hey @j-schilling, just add the `key` prop here. You're already using `index` so you can use that as the value for it, or use `id`, as the array entries do have unique IDs (`key={index}` / `key={id}`).